api/samples/bbbuf.c

```diff
@@ -67,6 +67,8 @@ event_app_instruction(void *drcontext, void *tag, instrlist_t *bb, instr_t *inst
     /* We need a 2nd scratch reg for several operations on AArch32 and AArch64 only. */
     reg_id_t reg2 = DR_REG_NULL;
 
+    drmgr_disable_auto_predication(drcontext, bb);
```

Comment:
Since these are samples showing newbies how to write a client, it's worth a comment explaining why we're doing this, here and in the other samples.